pkg/ccl/crosscluster/logical/create_logical_replication_stmt.go

```diff
@@ -211,16 +210,9 @@ func createLogicalReplicationStreamPlanHook(
 
 		sourceTypes := make([]*descpb.TypeDescriptor, len(spec.TypeDescriptors))
 		for i, desc := range spec.TypeDescriptors {
-			// Until https://github.com/cockroachdb/cockroach/issues/132164 is resolved,
-			// we cannot allow user-defined types on the SQL ingestion path.
```

Comment:
i think this little block of code should be in the previous commit.

Reply:
done